packages/react-mongo/react-mongo.ts

```diff
@@ -32,9 +25,9 @@ const useSubscriptionClient = (factory: () => Meteor.SubscriptionHandle | void,
     // it stops the inner one.
     const computation = Tracker.nonreactive(() => (
       Tracker.autorun(() => {
-        refs.handle = factory()
-        if (!refs.handle) return
-        if (refs.updateOnReady && refs.handle.ready()) {
+        refs.subscription = factory()
+        if (!refs.subscription) return
+        if (refs.updateOnReady && refs.subscription.ready()) {
```

Comment:
does that really work ?
updateOnReady starts as false, so the tracker computation doesn't run subscription.ready() and thus doesn't register it as a reactive dep.
Then if the user calls the ready() func we give him, updateOnReady is changed to true, but that doesn't make the tracker computation rerun and become reactive to the ready state change ?

Reply:
Ah, good point. I actually swapped these around (`if (refs.subscription.ready() && refs.updateOnReady)`) to fix this, but I took it out of this PR as to not change too much. Do you think that would work if it is swapped?

Reply:
yep, sounds like that should solve it.
the "become reactive once you actually read it" pattern is not super intuitive IMO :-D 
as discussed in the main PR, I tend to think leaving ready(reactivity out of the hook might be simpler...

Reply:
Agreed. I'm working on a PR now, will let you know when it's ready so you can review.